Base/res/devel/templates/cpp-gui/main.cpp

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <LibGUI/Application.h>
+#include <LibGUI/Window.h>
+#include <LibGUI/Button.h>
+#include <LibGUI/MessageBox.h>
+
+int main(int argc, char** argv)
+{
+    auto app = GUI::Application::construct(argc, argv);
+
+    auto window = GUI::Window::construct();
+    window->set_title("Hello friends!");
+    window->resize(200, 100);
+
+    auto button = GUI::Button::construct();
+    button->set_text("Click me!");
+    button->on_click = [&]() {
```

Comment:
Does this actually work? I think this needs to be:

```suggestion
    button->on_click = [&](auto) {
```

As `on_click` will receive modifiers:

https://github.com/SerenityOS/serenity/blob/13d7c09125f8eec703d0a43a9a87fc8aa08f7319/Userland/Libraries/LibGUI/Button.h#L50

Reply:
I have tested and interestingly it does, I'll make this change either way in the interest of the templates serving as good examples for developers :)